calculateFrequency.js

```diff
@@ -1,8 +1,6 @@
 // Problem 2: Complete the calculateFrequency function that takes lowercase string as input and returns frequency of all english alphabet. (using only array, no in-built function)
 function calculateFrequency(string) {
-  var length;
-
-  length = string.length;
+  var length = string.length;
   var output = {};
 
   for (var iteration = 0; iteration < length; iteration++) {
```

Comment:
@abansal26 name the variables according to what they do or are meant for.
For example here the for loop does the iteration and not the variable, the variable instead defines the current index for the running iteration. Keep this in mind for all the variables in this PR and in future.